test/absinthe/schema/sdl_render_test.exs

```diff
@@ -0,0 +1,283 @@
+defmodule SdlRenderTest do
+  use ExUnit.Case
+
+  defmodule TestSchema do
+    use Absinthe.Schema
+
+    # Working based on import_sdl_test.exs
+
+    @sdl """
+    type User {
+      name: String!
+    }
+
+    "One or the other"
+    union SearchResult = Post | User
+
+    type Query {
+      echo(
+        category: Category!
+        "The number of times"
+        times: Int
+      ): [Category]!
+      posts: Post
+      search(
+        query: String!
+      ): [SearchResult]
+    }
+
+    \"\"\"
+    A submitted post
+    Multiline description
+    \"\"\"
+    type Post {
+      title: String!
+    }
+
+    "Simple description"
+    enum Category {
+      NEWS
+      OPINION
+    }
+    """
+    import_sdl @sdl
+    def sdl, do: @sdl
+  end
+
+  import Inspect.Algebra
+
+  @moduledoc """
+  https://github.com/graphql/graphql-js/blob/master/src/utilities/schemaPrinter.js
+
+  skips:
+    - built in scalars.. String Int Float Boolean ID
+    - introspection types.. `__Type`
+
+  issues:
+    - schema definition order is not respected?
```

Comment:
I'm noticing that the order of items defined in the original SDL is not being respected any longer once we get the results back from the introspection query..